src/web/aws-cloudwatch/KinesisSetup.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import { Button, Col } from 'react-bootstrap';
+import { Input } from 'components/bootstrap';
+
+export default class KinesisSetup extends Component {
+  static propTypes = {
+    onSubmit: PropTypes.func.isRequired,
+    onChange: PropTypes.func.isRequired,
+    getValue: PropTypes.func.isRequired,
+  }
+
+  render() {
+    const { getValue, onChange, onSubmit } = this.props;
+
+    return (
+      <form onSubmit={onSubmit}>
+        <Col md={9} mdOffset={6}>
+          <h2>Create Kinesis Stream</h2>
+          <p>We're going to get started setting up your Kinesis Stream, just give us a name and choose the related CloudWatch Group. We'll handle the hard stuff!</p>
+
+          <Input
```

Comment:
The form itself looks different than other forms we have, but for me inputs in this case look way larger than any other content in the page, and they are also larger than inputs anywhere in the product. We have never been good in consistency, but I still think we should try to make this form more integrated in the style of the rest of the product.